Gemfile

```diff
@@ -8,7 +8,7 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", "~> 3.8.4"
```

Comment:
I'm a little nervous about this. I'm pretty sure newer Jekyll will break things. We should upgrade, but I feel like it's not this simple

Reply:
The changed applied to the Gemfile can be removed, but I wasnt unable to get the site to build otherwise, even with none of my own changes added. There was a conflict with the kramdown version that was segfaulting jekyll whenever I tried to run the site.

Reply:
What in particularly are you concerned about breaking? I ran a test site to confirm that the changes didn't accidentally impact anything else, and no issues caught my eye.

Reply:
If things built fine then that's fine, I just knew we were running quite an old version of Jekyll by this point and figured pinning was important

Reply:
The build happens in docker in OKD, so make sure it builds in the docker container

Reply:
Ok, will do. I'll reply to this thread once I've done so.